meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -0,0 +1,2 @@
+# XXX Drop the VM hack after taking care also of the non UEFI devices (those using U-Boot: edison and beaglebone)
+OSTRO_VM_IMAGE_TYPES = "dski.zip dsk.vdi.zip"
```

Comment:
On 18 March 2016 at 11:23, Patrick Ohly <notifications@github.com> wrote:

> In meta-ostro/conf/distro/include/ostroproject-ci.inc
> <https://github.com/ostroproject/meta-ostro/pull/50#discussion_r56629858>:
>
> > @@ -0,0 +1,2 @@
> > +# XXX Drop the VM hack after taking care also of the non UEFI devices (those using U-Boot: edison and beaglebone)
> > +OSTRO_VM_IMAGE_TYPES = "dski.zip dsk.vdi.zip"
>
> Typo: s/dski.zip/dsk.zip/
>

​fixed, thanks.​


> @kad <https://github.com/kad> How do you want the CI to use include
> files? Include both "ostro-development.inc" and "ostroproject-ci.inc", or
> just "ostroproject-ci.inc"? I had the impression that it is the latter, in
> which case this file here must include
> ​​
> ostro-development.inc.
>
>
​Doesn't CI need to produce also images based on ostro-os-production.inc?​


​Wouldn't the inclusion here of ​​ostro-development.inc here prevent those?




-- 
igor